libraries/tools/kotlin-gradle-plugin-integration-tests/src/test/kotlin/org/jetbrains/kotlin/gradle/native/CocoaPodsIT.kt

```diff
@@ -1251,7 +1251,7 @@ class CocoaPodsIT : BaseGradleIT() {
                                 set -ev
                                 REPO_ROOT="${'$'}PODS_TARGET_SRCROOT"
                                 "${'$'}REPO_ROOT/../gradlew" -p "${'$'}REPO_ROOT" :kotlin-library:syncFramework \
-                                    -Pkotlin.native.cocoapods.target=${'$'}KOTLIN_TARGET \
+                                    -Pkotlin.native.cocoapods.targets=${'$'}KOTLIN_TARGET \
```

Comment:
Renaming this property makes the plugin incompatible with Xcode projects configured by older versions of the plugin. Could you support both property names or at least don't rename it?

Reply:
I decided to revert it back to singular form